source/_patterns/01-atoms/icon/scss/_icon.scss

```diff
@@ -3,6 +3,9 @@
 ///
 
 @import 'icon-mixins';
+@import '~font-awesome/scss/mixins';
+@import '~font-awesome/scss/variables';
```

Comment:
just noting these files don't have printing output, totally fine to include here.